app/javascript/mastodon/components/status_list.js

```diff
@@ -1,11 +1,31 @@
+import { debounce } from 'lodash';
 import React from 'react';
 import ImmutablePropTypes from 'react-immutable-proptypes';
 import PropTypes from 'prop-types';
 import StatusContainer from '../containers/status_container';
 import ImmutablePureComponent from 'react-immutable-pure-component';
+import LoadMore from './load_more';
 import ScrollableList from './scrollable_list';
 import { FormattedMessage } from 'react-intl';
 
+class LoadGap extends ImmutablePureComponent {
+
+  static propTypes = {
+    disabled: PropTypes.bool,
+    maxId: PropTypes.string,
```

Comment:
I think it's possible to use min_id now too?

Reply:
It decides that a gap is filled by looking for statuses duplicate in the fetched list and the local list. `min_id` is omitted to bring such duplicate statuses.

Reply:
Ah, I think we didn't expose it in the API and there's no equivalent yet. Still, I wonder if min_id is necessary for the gap to function properly?